docs/notebooks/LangCheck Quickstart.ipynb

```diff
@@ -1,27 +1,13 @@
 {
```

Comment:
Just curious, what's this quickstart change for? I compared the previous & current notebooks with my eyes but couldn't find any significant difference.

Reply:
The text diff is a bit weird, but I deleted the second empty cell in the notebook. You can see the empty cell here: https://github.com/citadel-ai/langcheck/blob/2de006353419b5bef4d76666fa83f57a68fccd45/docs/notebooks/LangCheck%20Quickstart.ipynb

Reply:
Got it, feel free to merge then!